src/main/java/site/facade/VideoSanitizerService.java

```diff
@@ -0,0 +1,11 @@
+package site.facade;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class VideoSanitizerService {
```

Comment:
This looks like Util method for me. Also it doesn't work with a Video actually, it is more like working with a String name.